defaults/main.yml

```diff
@@ -1,6 +1,7 @@
 ---
 prometheus_version: 2.15.1
 prometheus_binary_local_dir: ''
+prometheus_checksums: []
```

Comment:
This needs an entry in README.md. Especially explanation on what are the possible entries and how this can be used when `prometheus_binary_local_dir` is set.